app/services/platform/user_datastore_adapter.rb

```diff
@@ -27,7 +27,8 @@ def load_data
       response = request(:get, url, {}).body['payload']
 
       JSON.parse(data_encryption.decrypt(response))
-    rescue Platform::ResourceNotFound
+    rescue Platform::ResourceNotFound => e
+      Sentry.capture_exception(e)
```

Comment:
Humm when the user clicks starts for the first time one request is 404. I think this will be sent to the sentry once per user.

Reply:
ah yeah, had forgot about that one. i'll take it out for now